geopandas/tests/test_pandas_methods.py

```diff
@@ -582,3 +583,34 @@ def test_preserve_flags(df):
 
     with pytest.raises(ValueError):
         pd.concat([df, df])
+
+
+def test_concat(df):
```

Comment:
It seems the current `concat` tests are in `test_merge.py`. Can you move it there? (it's certainly not straightforward to know where best to put tests in such an existing package!)